### ISSUE TITLE
random line of tag/class/id/gibberish doesn't throw error

### ISSUE THREAD

**Issue by mmase** [sasstools/sass-lint#916]
I would expect this to throw some sort of linting error, though I don't think it's captured in sass-lint:

```sass
.element {
  background: #ff0;
}

div  // doesn't throw a lint warning

#myElement {
  color: #f00;
}
```

**Comment by DanPurdy** [comment]
Believe it or not, it's actually valid Sass. You can try it here http://www.sassmeister.com/

It'll be compiled as `div #myElement` Not sure what kind of lint warning we could throw for that as we don't really have a specific rule looking for extra line breaks in selectors.

You can see how our AST reports it below

```
{
  "type": "stylesheet",
  "content": [
    {
      "type": "ruleset",
      "content": [
        {
          "type": "selector",
          "content": [
            {
              "type": "typeSelector",
              "content": [
                {
                  "type": "ident",
                  "content": "div"
                }
              ]
            },
            {
              "type": "space",
              "content": "\n\n"
            },
            {
              "type": "id",
              "content": [
                {
                  "type": "ident",
                  "content": "myElement"
                }
              ]
            }
          ]
        },
        {
          "type": "space",
          "content": " "
        },
        {
          "type": "block",
          "content": [
            {
              "type": "space",
              "content": "\n  "
            },
            {
              "type": "declaration",
              "content": [
                {
                  "type": "property",
                  "content": [
                    {
                      "type": "ident",
                      "content": "color"
                    }
                  ]
                },
                {
                  "type": "propertyDelimiter",
                  "content": ":"
                },
                {
                  "type": "space",
                  "content": " "
                },
                {
                  "type": "value",
                  "content": [
                    {
                      "type": "color",
                      "content": "f00"
                    }
                  ]
                }
              ]
            },
            {
              "type": "declarationDelimiter",
              "content": ";"
            },
            {
              "type": "space",
              "content": "\n"
            }
          ]
        }
      ]
    }
  ]
}
```

**Comment by mmase** [comment]
Interesting, I was testing with http://beautifytools.com/scss-compiler.php and it didn't compile that way. But I confirmed that it does as sassmeister says it will. Sorry for the false alarm!
